arq/worker.py

```diff
@@ -275,11 +278,11 @@ def run(self) -> None:
 
             if self.burst:
                 if 0 <= self.max_burst_jobs <= self._jobs_started():
-                    await asyncio.gather(*self.tasks)
+                    await asyncio.gather(*list(self.tasks.values()))
```

Comment:
```suggestion
                    await asyncio.gather(*self.tasks.values())
```
I doubt list is required here.